git_machete/github.py

```diff
@@ -214,6 +214,18 @@ def __fire_github_api_request(method: str, path: str, token: Optional[str], requ
                 f'`{method} {url}` request ended up in 404 response from GitHub. A valid GitHub API token is required.\n'
                 f'Provide a GitHub API token with `repo` access via one of the: {get_github_token_possible_providers()} '
                 'Visit `https://github.com/settings/tokens` to generate a new one.')  # TODO (#164): make dedicated exception here
+        elif err.code == http.HTTPStatus.TEMPORARY_REDIRECT:
+            current_repo_and_org = get_repo_and_org_names_by_id(err.headers['Location'].split('/')[4])
+            new_path = f'/repos/{current_repo_and_org}/{"/".join(path.split("/")[4:])}'
+            print(fmt(f'<yellow>\nGitHub API returned {err.code} HTTP status with error message: {err.reason}. '
```

Comment:
Maybe use `def warn`, we've got something like that in utils

Reply:
@PawelLipski Hmm, warn has red color which for me identifies with the error and since this is not an error (we can handle it) i wouldn't color it with red, wdyt?

Reply:
Hmmm actually... maybe change the color in `def warn` to orange? Yellow might be hard to spot on white-background terminals (pls check on your Mac, actually)

Reply:
@PawelLipski IMO yellow is nicely visible on white background 
<img width="1326" alt="image" src="https://user-images.githubusercontent.com/86779115/184841787-3542c653-0283-47d2-85a9-015781361c86.png">
so i would leave it as it is, wdyt?

Reply:
Hmmm okay... still, I'd insist making the format of all warnings consistent. I'd rather use `def warn` (possibly changed to get rid of red color as you suggested ☝🏻) here as well.

Reply:
Alright, i'll update the `warn` in separate PR, here's how it will look like after: 
<img width="1366" alt="image" src="https://user-images.githubusercontent.com/86779115/184889842-c308e72f-2c51-42a4-a772-370431c8aae6.png">
@PawelLipski ok?

Reply:
Okay, sure